docs/fr/week06/06-2.md

```diff
@@ -492,7 +492,7 @@ L'idée d'un réseau mémoire est qu'il y a deux parties importantes dans votre
 Pour un réseau mémoire, il y a une entrée au réseau, $x$ (pensez à cela comme une adresse de la mémoire), et comparez ce $x$ avec les vecteurs $k_1, k_2, k_3, \cdots$ ("clés") à travers un produit scalaire. En les faisant passer par une softmax, on obtient un tableau de nombres dont la somme est égale à un. Et il y a un ensemble d'autres vecteurs $v_1, v_2, v_3, \cdots$ ("valeurs"). Multipliez ces vecteurs par les scalaires provenant du softmax et additionnez ces vecteurs vous donne le résultat (notez la ressemblance avec le mécanisme d'attention).
 
 <center>
-<img src="{{site.baseurl/images/week06/06-2/MemoryNetwork1.png" height="300px"/><br>
+<img src="{{site.baseurl}}/images/week06/06-2/MemoryNetwork1.png" height="300px"/><br>
```

Comment:
This was the error preventing the build from completing locally.

Reply:
Yeah, we've fixed it already.